### DISCUSSION TITLE
Any method to subscribe to validation result ?

### DISCUSSION THREAD

**Discussion by jlopezxs** [react-hook-form/react-hook-form#7996] [Q&A]
Hi! I'm working on an error validations event logger. I've been looking at the internal code but I don't see any way to make it fit the on Blur mode.

I tried the handleSubmit onInvalid but doesn't work for me I need it by every change onBlur.
I tried to decorate the register onBlur method but the errors are not updated when the event is fired.
I've tried using the resolver but the original validate logic is not exported to call it and maintain the same behaiviour.

Any idea how to have something like "handleValidateResult" ?

**Comment by Moshyfawn** [comment]
Does the form element's `onBlur` event help? See #7613 for details
